Cargo.lock

```diff
@@ -6986,3 +7260,8 @@ dependencies = [
  "podio",
  "time",
 ]
+
+[[patch.unused]]
+name = "backtrace"
+version = "0.3.40"
+source = "git+https://github.com/MeFisto94/backtrace-rs?branch=fix-strtab-freeing-crash#91a0aa4a5d649151878cddd883b92ba7057ca41c"
```

Comment:
After rebased, I updated `color-backtrace` to be `{ version = "0.3", optional = true }` and also added a feature for it.

But, I will see this warning with running `./mach build`:

```
→ ./mach build -r
    Blocking waiting for file lock on package cache
warning: Patch `backtrace v0.3.40 (https://github.com/MeFisto94/backtrace-rs?branch=fix-strtab-freeing-crash#91a0aa4a)` was not used in the crate graph.
Check that the patched package version and available features are compatible
with the dependency requirements. If the patch has a different version from
what is locked in the Cargo.lock file, run `cargo update` to use the new
version. This may also occur with an optional dependency that is not enabled.
```

then, after compiling most of crates, it will fail at

```
error[E0432]: unresolved import `backtrace::PrintFmt`
  --> ports/glutin/backtrace.rs:13:36
   |
13 | use backtrace::{BytesOrWideString, PrintFmt};
   |                                    ^^^^^^^^ no `PrintFmt` in the root

error[E0433]: failed to resolve: could not find `BacktraceFmt` in `backtrace`
  --> ports/glutin/backtrace.rs:45:36
   |
45 |             let mut f = backtrace::BacktraceFmt::new(fmt, mode, &mut p);
   |                                    ^^^^^^^^^^^^ could not find `BacktraceFmt` in `backtrace`

error[E0425]: cannot find function `resolve_frame_unsynchronized` in crate `backtrace`
  --> ports/glutin/backtrace.rs:58:28
   |
58 |                   backtrace::resolve_frame_unsynchronized(frame, |symbol| {
   |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: a function with a similar name exists: `resolve_unsynchronized`
   |
  ::: /Users/cybai/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.15/src/symbolize/mod.rs:59:1
   |
59 | / pub unsafe fn resolve_unsynchronized<F>(addr: *mut c_void, mut cb: F)
60 | |     where F: FnMut(&Symbol)
61 | | {
62 | |     resolve_imp(addr as *mut _, &mut cb)
63 | | }
   | |_- similarly named function `resolve_unsynchronized` defined here

error: aborting due to 3 previous errors

Some errors have detailed explanations: E0425, E0432, E0433.
For more information about an error, try `rustc --explain E0425`.
error: could not compile `servo`.
```

I think it might be related to this accidentally unused crate but I'm not sure how I can only get the lockfile updated for `color-backtrace`. 🤔

Reply:
Thanks @Eijebong !

This is fixed after running `cargo update -p backtrace --precise 0.3.40`!